src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -12,13 +12,32 @@
     [status-im.constants :as constants]
     [status-im.contexts.wallet.common.utils :as utils]
     [status-im.contexts.wallet.create-account.style :as style]
-    [status-im.feature-flags :as ff]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]
     [utils.responsiveness :refer [iphone-11-Pro-20-pixel-from-width]]
     [utils.security.core :as security]
     [utils.string]))
 
+(defn prepare-new-keypair
+  [{:keys [new-keypair address account-name account-color emoji derivation-path]}]
+  (assoc
```

Comment:
perhaps a slight readability improvement could be to move this into a let var
```
(let [keypair-with-info   (assoc new-keypair
          :name         (:keypair-name new-keypair)
          :key-uid      (:keyUid new-keypair)
          :type         :seed
          :derived-from address)]
(assoc
   keypair-with-info
   :accounts
   [{:keypair-name (:keypair-name new-keypair)
     :key-uid      (:keyUid new-keypair)
     :seed-phrase  (:mnemonic new-keypair)
     :public-key   (:publicKey new-keypair)
     :name         account-name
     :type         :seed
     :emoji        emoji
     :colorID      account-color
     :path         derivation-path
     :address      (:address new-keypair)}])))
 ```

Reply:
I guess a single `assoc` should suffice:

```clojure
(assoc new-keypair
          :name         (:keypair-name new-keypair)
          :key-uid      (:keyUid new-keypair)
          :type         :seed
          :derived-from address
          :accounts [{:keypair-name (:keypair-name new-keypair)
                      :key-uid      (:keyUid new-keypair)
                      :seed-phrase  (:mnemonic new-keypair)
                      :public-key   (:publicKey new-keypair)
                      :name         account-name
                      :type         :seed
                      :emoji        emoji
                      :colorID      account-color
                      :path         derivation-path
                      :address      (:address new-keypair)}])

```